src/ui/hooks/users.ts

```diff
@@ -31,44 +29,32 @@ export enum Nag {
 }
 
 export function useGetUserInfo() {
-  const userId = getUserId();
   const { data, loading, error } = useQuery(
     gql`
-      query GetUser($userId: uuid!) {
-        users_by_pk(id: $userId) {
-          invited
+      query GetUser {
+        viewer {
           email
-          invitations {
-            pending
-          }
-          recordings_aggregate {
-            aggregate {
-              count
-            }
-          }
           internal
           nags
+          recordings {
+            totalCount
```

Comment:
This doesn't exist in the API right now, but hopefully I can remove our need for `authoredRecordingCount` entirely, then I'll just drop this part of the query.